### DISCUSSION TITLE
Exit codes

### DISCUSSION THREAD

**Discussion by cigzigwon** [CMSgov/price-transparency-guide-validator#76] [General]
Where do we reference. the exit codes? Specifically error #2? I think I caused it but would be nice to know.

**Comment by shaselton-usds** [comment]
The current exit codes are pretty simplistic at the moment. Exit code 0 = success (non validation error) and exit code 1 = validation error.

Would additional codes be helpful? If so, what are you thinking?

**Comment by cigzigwon** [comment]
At one point I received an exit code of `2` but in retrospect, I think I yielded this code via the Erlang interface that is talking to the Validator. So I don't think the code was actually coming from the validator in my case rather Erlang itself calling the program via npx bootstrapping. In my case 0 vs 1 aka PASS vs FAIL is fine! I think if others see error 2 it would be best to check the way they interface with the program.

And to add clarification I'm not using the validator from a shell but rather need to interface with it via Elixirlang which uses Erlang pipes to interface w/npx so that the program doesn't need to be installed globally.
